tox.ini

```diff
@@ -42,106 +42,104 @@
     python-framework_ariadne-py37-ariadne{0011,0012,0013},
     python-framework_bottle-py27-bottle{0008,0009,0010},
-    python-framework_bottle-{py27,py36,py37,py38,py39,pypy36}-bottle{0011,0012},
+    python-framework_bottle-{py27,py37,py38,py39,pypy36}-bottle{0011,0012},
     python-framework_bottle-py310-bottle0012,
     python-framework_bottle-pypy-bottle{0008,0009,0010,0011,0012},
-    python-framework_cherrypy-{py36,py37,py38,py39,py310,pypy36}-CherryPy18,
-    python-framework_cherrypy-{py36,py37}-CherryPy0302,
+    python-framework_cherrypy-{py37,py38,py39,py310,pypy36}-CherryPy18,
+    python-framework_cherrypy-{py37}-CherryPy0302,
     python-framework_cherrypy-pypy36-CherryPy0303,
     python-framework_django-{pypy,py27}-Django0103,
     python-framework_django-{pypy,py27,py37}-Django0108,
     python-framework_django-{py39}-Django{0200,0201,0202,0300,0301,latest},
-    python-framework_django-{py36,py37,py38,py39,py310}-Django0302,
-    python-framework_falcon-{py27,py36,py37,py38,py39,pypy,pypy36}-falcon0103,
-    python-framework_falcon-{py36,py37,py38,py39,py310,pypy36}-falcon{0200,master},
-    python-framework_fastapi-{py36,py37,py38,py39,py310},
+    python-framework_django-{py37,py38,py39,py310}-Django0302,
+    python-framework_falcon-{py27,py37,py38,py39,pypy,pypy36}-falcon0103,
+    python-framework_falcon-{py37,py38,py39,py310,pypy36}-falcon{0200,master},
+    python-framework_fastapi-{py37,py38,py39,py310},
     python-framework_flask-{pypy,py27}-flask0012,
-    python-framework_flask-{pypy,py27,py36,py37,py38,py39,py310,pypy36}-flask0101,
+    python-framework_flask-{pypy,py27,py37,py38,py39,py310,pypy36}-flask0101,
     python-framework_flask-{py37,py38,py39,py310,pypy37}-flask{latest,master},
-    python-framework_graphene-{py36,py37,py38,py39,py310}-graphenelatest,
-    python-framework_graphene-{py27,py36,py37,py38,py39,pypy,pypy36}-graphene{0200,0201},
+    python-framework_graphene-{py37,py38,py39,py310}-graphenelatest,
+    python-framework_graphene-{py27,py37,py38,py39,pypy,pypy36}-graphene{0200,0201},
     python-framework_graphene-py310-graphene0201,
-    python-framework_graphql-{py27,py36,py37,py38,py39,py310,pypy,pypy36}-graphql02,
-    python-framework_graphql-{py36,py37,py38,py39,py310,pypy36}-graphql03,
+    python-framework_graphql-{py27,py37,py38,py39,py310,pypy,pypy36}-graphql02,
+    python-framework_graphql-{py37,py38,py39,py310,pypy36}-graphql03,
     python-framework_graphql-py37-graphql{0202,0203,0300,0301,0302,master},
-    grpc-framework_grpc-{py27,py36}-grpc0125,
-    grpc-framework_grpc-{py36,py37,py38,py39,py310}-grpclatest,
+    grpc-framework_grpc-{py27}-grpc0125,
+    grpc-framework_grpc-{py37,py38,py39,py310}-grpclatest,
     python-framework_pyramid-{pypy,py27,py38}-Pyramid0104,
-    python-framework_pyramid-{pypy,py27,pypy36,py36,py37,py38,py39,py310}-Pyramid0110-cornice,
+    python-framework_pyramid-{pypy,py27,pypy37,py38,py39,py310}-Pyramid0110-cornice,
```

Comment:
Is this intentionally upgraded to pypy 3.7?

Reply:
From the most recent version of main this was added "temporarily disabling pypy36 on pyramid master" as a comment regarding that change so I believe it is.